readme.md

```diff
@@ -10,17 +10,15 @@ npm install --save-dev assemblyscript as-lunatic
 
 Then, install [lunatic](https://github.com/lunatic-solutions/lunatic).
 
-Next, modify your asconfig to include as-lunatic as a `lib`. If the `lib` option already exists, add it to the array.
+Next, modify your asconfig to extend as-lunatic,
 
 ```json
 {
-  "options": {
-    "lib": ["./node_modules/as-lunatic/assembly"]
-  }
+  "extends": "as-lunatic/asconfig.json"
 }
 ```
 
-Finally, import wasi into your module entry point.
+Finally, import Wasi into your module entry point.
```

Comment:
For now, importing `wasi` implies `--explicitStart` which breaks stuff.

It's best to `export function _start(): void {}` instead of using `import "wasi"` at the moment.

Reply:
True maybe we should use main and then have an alias to rename it to `_start` to make it more like Rust, C, etc.